tools/Makefile

```diff
@@ -31,10 +31,10 @@ nkf:
 	cd nkf; tar zxvf nkf-2.1.4.tar.gz; cd nkf-2.1.4; $(MAKE) prefix=.
 
 venv/lib/python2.7/site-packages/torch: venv/bin/activate
-	. venv/bin/activate; pip install pip --upgrade; pip install torch==0.4.0
+	. venv/bin/activate; pip install pip --upgrade; pip install torch==0.4.1
 
 warp-ctc: venv/lib/python2.7/site-packages/torch
-	git clone https://github.com/jnishi/warp-ctc.git
+	git clone https://github.com/SeanNaren/warp-ctc.git
```

Comment:
Use  `follows-upstream` branch of https://github.com/jnishi/warp-ctc of supports  because it supports pytorch 0.4.1 and https://github.com/espnet/espnet/pull/307.

Reply:
When this PR is merged, I will fix `master` branch to support pytorch 0.4.1.